### PR TITLE
Add EKCO version 0.19.0 to enable privileges to approve CSRs

### PR DESCRIPTION
**Pull request by jala-dx**
#### What type of PR is this?
Creates a new version of EKCO addon that enables privileges to approve CSRs.
These CSRs are generated to let kubelet to use the /etc/kubernetes/pki/ca.crt as its Certificate Authority instead of using self-signed certs.

#### Which issue(s) this PR fixes:

Fixes #
SC-31335

#### Special notes for your reviewer:
Please review rbac.yaml

#### Does this PR introduce a user-facing change?
No

```release-note

Add EKCO version 0.19.0 to enable privileges to approve CSRs

```

### CODE REVIEW

File: addons/ekco/0.19.0/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- `ekco-should-disable-clear-nodes`: Don't force delete pods on unreachable stuck in the terminating state
+- `ekco-disable-should-maintain-rook-storage-nodes`: Whether to maintain the list of nodes to use in the CephCluster config.
+   This also enables control of the replication factor of ceph pools, scaling up and down with the number of nodes in the cluster.
+- `ekco-disable-should-install-reboot-service`: Whether to disable the reboot service, responsible for graceful node termination.
+- `rook-version`: Version of Rook to manage
+- `rook-priority-class`: Name of priority class to add to Rook 1.0 Deployments and DaemonSets. Will be created if not found.
+- `enable-internal-load-balancer`: Run haproxy on all nodes and forward to all Kubernetes API server pods
+- `host-task-image`: The image to use for host tasks, such as updating the haproxy configuration on all nodes.
+- `host-task-namespace`: The namespace host task pods will run in.
+- `pod-image-overrides`: List of images to be replaced in pod specs
```
Comment (emosbaugh) [inline addons/ekco/0.19.0/README.md:23]: add a flag to optionally enable auto approve csrs from kubelet feature

**Review by emosbaugh — CHANGES_REQUESTED** [review]
Please add a flag and document this new functionality of ekco

File: scripts/install.sh [hunk 2]
```diff
@@ -329,7 +329,8 @@ function kurl_config() {
         --from-literal=upload_certs_expiration="$CERT_KEY_EXPIRY" \
         --from-literal=service_cidr="$SERVICE_CIDR" \
         --from-literal=pod_cidr="$POD_CIDR" \
-        --from-literal=kurl_install_directory="$KURL_INSTALL_DIRECTORY_FLAG"
+        --from-literal=kurl_install_directory="$KURL_INSTALL_DIRECTORY_FLAG" \
+        --from-literal=kurl_cis_compliance="$CIS_COMPLIANCE"
```
Comment (emosbaugh) [inline scripts/install.sh:333]: see my comment here https://github.com/replicatedhq/ekco/pull/43#discussion_r830236527

**Review by emosbaugh — CHANGES_REQUESTED** [review]

**Review by emosbaugh — APPROVED** [review]

File: scripts/install.sh (hunk 2 above)
Comment (jala-dx) [reply scripts/install.sh:333]: removed and added it to ekco configmap

**Pull request merged**